src/components/usa-button/content/usa-button~outline-inverse.json

```diff
@@ -1,5 +1,6 @@
 {
```

Comment:
![image](https://user-images.githubusercontent.com/93996430/151075348-6c8bf545-140a-4f16-a35c-3bbf2b2930a2.png)
The is_demo:false setting appears to be broken (see screenshot)